### DISCUSSION TITLE
Get rid of additional "caption" for subtables in revealjs presentation inside custom division

### DISCUSSION THREAD

**Discussion by dmkaplan2000** [quarto-dev/quarto-cli#5706] [Q&A] [answered]
### Description

I am trying to add to a revealjs quarto slide a table with subtables. I wanted the tables in a small font to make them appear on the page, so I placed them inside a custom Div, but this makes it ask for an additional caption for each subtable (i.e., `?(caption)` appears below each subtable). Can this be fixed or should I try a different approach (e.g., using columns)? Example document below...

````qmd
---
title: "Untitled"
format: revealjs
---

## Subtable captions?

```{css}
.small-txt {
font-size: 30px !important;
}
```

:::{.small-txt}
```{r}
#| label: tbl-subtables
#| tbl-cap: "Some table"
#| tbl-subcap: 
#|   - "Tab1"
#|   - "Tab2"
#| layout-ncol: 2
df = data.frame(id=letters[1:3],vals=rnorm(3))

knitr::kable(df)
knitr::kable(df)
```
:::

````

**Comment by cscheid** [comment]
I don't think there's a clean solution right now, unfortunately, but a more consistent treatment of subtables, their captions, and their styles is on the [1.4 crossref epic](https://github.com/quarto-dev/quarto-cli/issues/4944).

**Comment by mcanouil** [comment, marked as answer]
Here the recommended/proper syntax:

<table>
<tr><th>Quarto document</th><th>HTML</th></tr>
<tr>
<td>

````qmd
---
title: "Untitled"
format: revealjs
include-in-header:
  - text: |
      <style>
      .small-txt {
        font-size: 30px !important;
        text-align: center;
      }
      </style>
---

## Subtable captions?

```{r}
#| classes: small-txt
#| label: tbl-subtables
#| tbl-cap: "Some table"
#| tbl-subcap: 
#|   - "Tab1"
#|   - "Tab2"
#| layout-ncol: 2
df <- data.frame(id = letters[1:3], vals = rnorm(3))

knitr::kable(df)
knitr::kable(df)
```

````

</td>
<td>
<img width="728" alt="image" src="https://github.com/quarto-dev/quarto-cli/assets/8896044/b921d7cf-c92e-48bf-9e9b-2bfe9e309f45">
</td>
</tr>
</table>

**Reply by cscheid** [reply]
TIL about `classes`!

**Comment by dmkaplan2000** [comment]
`classes` worked perfectly. Thanks for the help.
